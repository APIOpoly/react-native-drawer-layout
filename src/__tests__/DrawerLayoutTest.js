/* global test, expect */
import {
  View,
} from 'react-native';
import React from 'react'; // eslint-disable-line
import DrawerLayout from '../DrawerLayout';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('it renders correctly', () => {
  const tree = renderer.create(
    <DrawerLayout
      drawerPosition="left"
      renderNavigationView={() => <View />}>
      <View />
    </DrawerLayout>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
