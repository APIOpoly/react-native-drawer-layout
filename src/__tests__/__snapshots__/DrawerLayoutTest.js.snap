exports[`test it renders correctly 1`] = `
<View
  onMoveShouldSetResponder={[Function]}
  onMoveShouldSetResponderCapture={[Function]}
  onResponderEnd={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderReject={[Function]}
  onResponderRelease={[Function]}
  onResponderStart={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  onStartShouldSetResponderCapture={[Function]}
  style={
    Object {
      "backgroundColor": "transparent",
      "flex": 1,
    }
  }>
  <View
    style={
      Object {
        "flex": 1,
      }
    }>
    <View />
  </View>
  <View
    style={
      Object {
        "backgroundColor": undefined,
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "top": 0,
        "transform": Array [
          Object {
            "translateX": -0,
          },
        ],
        "width": 0,
      }
    }>
    <View />
  </View>
</View>
`;
